llvm/include/llvm/IR/Intrinsics.td

```diff
@@ -1003,6 +1003,14 @@ def int_memset_inline
        NoCapture<ArgIndex<0>>, WriteOnly<ArgIndex<0>>,
        ImmArg<ArgIndex<3>>]>;
 
+// Memset variant that writes a given pattern.
```

Comment:
Comment what the operands are

Reply:
Added with inline comments (there's not a totally consistent pattern for describing args - many intrinsics have no description at all, but I see some other examples in the file using inline comments for the args)

Reply:
Really we ought to have tablegen mandatory doc strings